drivers/bluetooth/hci/h4.c

```diff
@@ -423,20 +419,35 @@ static int h4_send(struct net_buf *buf)
 	return 0;
 }
 
+/** Setup the HCI transport, which usually means to reset the Bluetooth IC
+  *
+  * @param dev The device structure for the bus connecting to the IC
+  *
+  * @return number of additional bytes to discard, negative error
+  * 	    value on failure
+  */
```

Comment:
Wouldn't it be good to put a prototype for `bt_hci_transport_setup()` and this documentation in a driver-agnostic header file? I suppose `include/drivers/bluetooth/hci_driver.h` would be the obvious choice.

Reply:
Agreed, will fix now